common/root/opt/helios/idlehook.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+# Originally sourced from Juno's internal Polaris.
+# Original Author: @ddesmond
+# Modified by: @aldmbmtl
+
+timeout=$(($IDLE_TIME * 60 * 1000))
+echo "Idle Detection running..." >>/tmp/idle-detect.log
+echo "Configured timeout: $IDLE_TIME" >>/tmp/idle-detect.log
+
```

Comment:
We need to figure out if there are local renders running as well. Polaris currently doesn't detect this.

Reply:
Sounds good, can we put in a ticket and point here for us to setup a  CPU Load setup? Costa ran into this for his local renders and I want to make sure we keep track of it.